### PR TITLE
healthcheck support in docker-compose configuration

### PR DESCRIPTION
**Pull request by StasShymov**
- Docker-compose `healthcheck` configuration support
  - Docker container wait timeout default value made configurable using `startContainerWaitTimeout` configuration option

### CODE REVIEW

**Comment by rohanKanojia** [comment]
@StasShymov : Hello, do you think changes in this pull request collide with https://github.com/fabric8io/docker-maven-plugin/pull/1736 (it's a draft PR that was created some time ago but author didn't get time to fix it) ?

**Comment by StasShymov** [comment]
@rohanKanojia yes, partially
Both are adding support for healthcheck configuration in docker compose.
But mine doesn't touch any existing `<run>` definitions, since there is already `<wait>`:
> Add <healthCheck> to <run> as known from <build> Add healthcheck to a compose file service definition

**Comment by StasShymov** [comment]
@rohanKanojia is there anything I can do to help this patch go through?

**Comment by rohanKanojia** [comment]
@StasShymov : Let me try to get this merged this weekend.

File: src/main/java/io/fabric8/maven/docker/access/ContainerCreateConfig.java [hunk 1]
```diff
@@ -3,20 +3,21 @@
...
 
 import java.io.FileNotFoundException;
 import java.io.FileReader;
 import java.io.IOException;
-import java.util.Enumeration;
-import java.util.List;
-import java.util.Map;
-import java.util.Properties;
-import java.util.Set;
+import java.util.*;
```
Comment (rohanKanojia) [inline src/main/java/io/fabric8/maven/docker/access/ContainerCreateConfig.java:14]: Could you please revert this wildcard import changes?

**Comment by rohanKanojia** [comment]
@StasShymov : Is it possible to write an integration test for this functionality? I see there are integration tests for docker-compose in `it/` folder .

**Comment by StasShymov** [comment]
@rohanKanojia I updated the one that already exists, does it look ok to you?

**Comment by StasShymov** [comment]
I confirmed that `it/docker-compose-dependon` is still green after my changes

**Comment by rohanKanojia** [comment]
@StasShymov : Thanks a lot!

**Pull request merged**